app/models/flexible_criterion.rb

```diff
@@ -3,7 +3,8 @@
 # Represents a flexible criterion used to mark an assignment that
 # has the marking_scheme_type attribute set to 'flexible'.
 class FlexibleCriterion < Criterion
-
+  extend ActionView::Helpers
```

Comment:
Some instances of the t helper work by using extend and some others by using include. I am guessing sometimes we use instance methods, and sometimes we use class methods under the hood. I will revert the Internationalization changes of this pull request.